### DISCUSSION TITLE
Does this library support DevExpress window form?

### DISCUSSION THREAD

**Discussion by nult2003** [FlaUI/FlaUI#478] [Q&A]
Currently, I'm going on the Devexpress application. I wonder whether this library supports this kind of application or not. I'm looking forward to you guys. 
Thanks

**Comment by kuskitkd** [comment]
Hi, 
I am using DevExpress WindowsForms as well to create some automation and, so far, I am seeing some things that do work and detect patterns, but for some others I am having issues. Have you been able to make any progress?

DevExpress Support mentioned using [WindowsFormsSettings.UseUIAutomation](https://docs.devexpress.com/WindowsForms/DevExpress.XtraEditors.WindowsFormsSettings.UseUIAutomation) but I am unsure of where/how should that be enabled, is that supposed to be on the main application or somewhere from the test project? It seems to be somewhat the WinForms equivalent to their WPF [ClearAutomationEventsHelper.IsEnabled = false;](https://docs.devexpress.com/WPF/DevExpress.Xpf.Core.ClearAutomationEventsHelper.IsEnabled); mentioned in the [Wiki](https://github.com/FlaUI/FlaUI/wiki/Common-Issues#when-using-devexpress-controls-some-things-like-tab-content-are-not-updated), but I have not been able to check it yet.

**Comment by Roemer** [comment]
There is also a `ClearAutomationEventsHelper.IsEnabled` which should be set to `false`. Maybe that is only for WPF applications.
